content/blogposts/2023/changes-to-the-sourcegraph-release-cadence.md

```diff
@@ -0,0 +1,42 @@
+---
+title: 'Changes to the Sourcegraph release cadence'
+authors:
+- name: Coury Clark
+  url: https://github.com/coury-clark
+  publishDate: 2023-02-22T10:00-07:00
+  description: 'A summary of upcoming changes to the Sourcegraph release cadence.'
+  tags: [blog]
+  slug: 'changes-to-the-sourcegraph-release-cadence'
+  published: false
+  heroImage: https://storage.googleapis.com/sourcegraph-assets/blog/conc-og-social.png
+  socialImage: https://storage.googleapis.com/sourcegraph-assets/blog/conc-og-social.png
+---
+
+The past year we has been focused on improving the experience of administrating and running Sourcegraph in a variety of ways. 
+We've made it easier than ever to get started with [Sourcegraph Cloud](https://docs.sourcegraph.com/cloud), as well as simplified the on-prem experience by launching [Sourcegraph AMI](https://docs.sourcegraph.com/admin/deploy/machine-images/aws-ami).
+
+Additionally, we took a look at our release cycle and found it is more difficult for customers to land on a stable and feature-rich version of Sourcegraph than we want. We didn't like that some users felt they had to selectively manage which versions are worth the effort to upgrade. We want every version of Sourcegraph to be extremely valuable, and extremely stable. Therefore, we are excited to continue to simplify
```

Comment:
Adding a suggestion for a slightly different word this. Feel free to accept/reject - just an option!

```suggestion
Additionally, we took a look at our release cycle and how frequently Sourcegraph users upgrade their instances. We found that many users selectively manage which versions of Sourcegraph they upgrade to versus which they skip. When customers land on a stable and feature-rich version of Sourcegraph, they tend to stick with that version for a while. 

We want every version of Sourcegraph to be extremely valuable and extremely stable. Therefore, we are excited to continue to simplify
```